recipes/pyfizi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyfizi" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a4b47cf057fce63e6d912f87d0e497065436f4b501698543863b5c82970c56f6
+
+build:
+  skip: True  # [win or py!=36]
```

Comment:
Is this package available for 2.7 and 3.7 and just not 3.6? (just curious)

Reply:
It's available only for 3.6. I wanted to support py >= 36, but the conda environment wouldn't resolve under 37 due to pandas-plink.

Reply:
Ah, I misread the statement :D 
Here's a PR to add Python 3.7 support for `pandas-plink`: https://github.com/conda-forge/pandas-plink-feedstock/pull/32

Reply:
Thanks for the help!